src/Component/Config/src/Exception/FileLoaderImportCircularReferenceException.js

```diff
@@ -6,6 +6,15 @@ const FileLoaderLoadException = Jymfony.Component.Config.Exception.FileLoaderLoa
  * @memberOf Jymfony.Component.Config.Exception
  */
 class FileLoaderImportCircularReferenceException extends FileLoaderLoadException {
+    /**
+     * Constructor.
+     *
+     * @param {*[]} resources
+     * @param {null|int} [code = null]
+     * @param {Exception} [previous]
+     *
+     * @inheritdoc
```

Comment:
The jsdoc block completely overrides the parent one. `@inheritdoc` should be removed here.